index.html

```diff
@@ -1 +1,146 @@
+                            <img src="./Assets/skincare.svg" alt="exhibition">
+                            <h3>Skin care</h3>
+                            <p>More than just skin deep, Vegan and cruelty-free</p>
+                        </article>
+                        <article class="programItem">
+                            <img src="./Assets/haircare.svg" alt="forum">
+                            <h3>Hair care</h3>
+                            <p>Say goodbye to bad chemicals and unhealthy ingredients.</p>
+                        </article>
+                        <article class="programItem">
+                            <img src="./Assets/workshop.svg" alt="workshop">
+                            <h3>Workshop</h3>
+                            <p>More panels and workshops than last year</p>
+                        </article>
+                        <article class="programItem">
+                            <img src="./Assets/edit-regular.svg" alt="ignite">
+                            <h3>Q & A</h3>
+                            <p>This is your chance to be part of the conversation.</p>
+                            
+                        </article>
+                    </div>
+                    <button class="joinButton mobile"><a href="#">Join Beauty Summit 2022</a></button>
+                    <a href="#" target="_blank" rel="noopener" class="joinLink desktop">See the whole program</a>
+                </div>
+            </section>
+            <section class="speakerSection" id="speakers">
+                <h2 class="speakerTitle">Featured Speakers</h2>
+                <div class="lineS"></div>  
+                <div id="jsSection"></div>
+                <button class="moreButton mobile">More<i class="fas fa-chevron-down icon"></i></button>
+            </section>
+            <section class="partnerSection desktop" id="sponsor">
+                <div class="partnerContainer">
+                    <h4>Partner</h4>
+                    <div class="line"></div>
+                    <div class="partnerList">
+                        <div class="partnerItem">
+                            <img src="./Assets/SNAANA-logo.png" alt="partner1">
+                        </div>
+                        <div class="partnerItem">
+                            <img src="./Assets/Google Logo.png" alt="partner2">
+                        </div>
+                        <div class="partnerItem">
+                            <img src="./Assets/makeupLogo.jpg" alt="partner3">
+                        </div>
+                        <div class="partnerItem">
+                            <img src="./Assets/Beauty-Brand-Logos-2.png" alt="partner4">
+                        </div>
+                        <div class="partnerItem">
+                            <img src="./Assets/Beauty-Brand-Logos-9.png" alt="partner5">
+                        </div>
+                </div>
+            </section>
```

Comment:
**OPTIONAL** - For improved SEO optimization I'd suggest you wrapped each of the sponsors in a `li` tag and then wrap all the `li` tags in a `ul` tag 👍 👍 

- Just as the previous reviewer stated and as I said above, please try as much as possible to stick to the general design layout, (background color, font color, font weight, font size e.t.c). This section of your page doesn't exactly match the given design template. Please try to fix it 🙏 🙏 

Your project(please pay attention to the header text color, and the color of the section background):
<img width="960" alt="partners" src="https://user-images.githubusercontent.com/73607512/196999921-01020c4a-01d3-41be-99aa-7721e2f17539.PNG">

Design template:
<img width="625" alt="partners behance" src="https://user-images.githubusercontent.com/73607512/197000063-6e9f0e75-7d90-45b9-bef6-83e667f5ba23.PNG">